sources/academy/platform/get_most_of_actors/actor_readme.md

```diff
@@ -5,93 +5,96 @@ sidebar_position: 2
+:::info Example of input schema
 
-6. **Input**
+Twitter Scraper has the following input options. Click on the [input tab](https://apify.com/quacker/twitter-scraper/input-schema) for more information.
 
-    - Refer to the input tab on Actor's detail page. If you like, you can add a screenshot showing the user what the input fields will look like.
-    - This is an example of how to refer to the input tab:
+:::
 
-    > Twitter Scraper has the following input options. Click on the [input tab](https://apify.com/quacker/twitter-scraper/input-schema) for more information.
+### Output Data
 
-7. **Output**
+Mention that the extracted dataset can be downloaded in various formats such as `JSON`, `HTML`,or `CSV`. Provide a simplified `JSON` dataset example to illustrate the output structure, check [this Actor](https://apify.com/compass/crawler-google-places#output-example) for an example.
 
-    - Mention "You can download the dataset extracted by (Actor name) in various formats such as JSON, HTML, CSV, or Excel.”
-    - Add a simplified JSON dataset example, like here: [apify.com/compass/crawler-google-places#output-example](https://apify.com/compass/crawler-google-places#output-example)
+### Tips & advanced options
 
-8. **Tips or Advanced options section**
-    - Share any tips on how to best run the Actor, such as how to limit compute unit usage, get more accurate results, or improve speed.
+Share any tips or advanced options for running the Actor more effectively, such as limiting compute unit usage, improving accuracy, or increasing speed.
 
-If you want some general tips on how to make GitHub README that stands out, check out these guides. Not everything in there will be suitable for an Apify Actor README, so you should cherry-pick what you like and use your imagination.
+If applicable, provide information on creating a tutorial for the Actor or any other resources that users might find helpful.
 
-<br />
+If you want some general tips on how to make GitHub README that stands out, check out these guides.
 
-[Build a Stunning README For Your GitHub Profile](https://towardsdatascience.com/build-a-stunning-readme-for-your-github-profile-9b80434fe5d7)
+- [Build a Stunning README For Your GitHub Profile](https://towardsdatascience.com/build-a-stunning-readme-for-your-github-profile-9b80434fe5d7)
+- [How to Create a Beautiful README for Your GitHub Profile](https://yushi95.medium.com/how-to-create-a-beautiful-readme-for-your-github-profile-36957caa711c)
 
-[How to Create a Beautiful README for Your GitHub Profile](https://yushi95.medium.com/how-to-create-a-beautiful-readme-for-your-github-profile-36957caa711c)
+Not everything in there will be suitable for an Actor README, so you should cherry-pick what you like and use your imagination.
```

Comment:
I think most of it won't be suitable, because those two articles talk about GitHub _profile_ README, where you present yourself, not a GitHub _repository_ README, where you present your project.

We could link to https://opensource.guide/starting-a-project/#writing-a-readme instead, which links https://www.makeareadme.com/ or https://gist.github.com/PurpleBooth/109311bb0361f32d87a2, but surely there are plenty of other articles which talk about how to make a good project README.